### ISSUE TITLE
TypeError: Cannot read property 'tag_name' of undefined

### ISSUE THREAD

**Issue by MisanthropicBit** [thinca/action-setup-vim#4]
Thanks for making this project :)

I have a small vim plugin that I am trying to test on Github Actions. However, I am getting the following error about [`tag_name` not being a property of undefined](https://github.com/MisanthropicBit/vim-numbers/runs/1563601733?check_suite_focus=true) using [this setup](https://github.com/MisanthropicBit/vim-numbers/blob/master/.github/workflows/ci.yml). Upon further investigation, I noticed that the README is using `version` to list things like `stable` and `head` and `vim_version` to list specific software versions such as `v8.2.0000`. However, the project's [CI test file](https://github.com/thinca/action-setup-vim/blob/master/.github/workflows/test.yml) seems to do this the other way around. This prompted me to do it this way instead but without any luck.

I have marked this as a question rather than an issue since I am most likely not using the action correctly but I would advise a change in the README to clarify things if this is indeed a mistake.

**Comment by thinca** [comment]
`vim_version` input does not accept `stable` because `stable` tag does not exist in `vim/vim` repository.

With your workflow yaml file:

```yaml
matrix:
  vim_version: ['head', 'stable']
  include:
            - vim_type: 'vim'
              vim_version: 'stable'
              version: 'v8.0.0000'
# (snip)
steps:
  uses: 'thinca/action-setup-vim@v1'
  with:
    vim_version: '${{ matrix.vim_version || matrix.version }}'
```

In this case, `matrix.vim_version` == `stable` and `matrix.version` == `v8.0.0000`.  `stable` is passed to action.

You should put `matrix.version` at first.

```yaml
    vim_version: '${{ matrix.version || matrix.vim_version }}'
```

---

Certainly, in CI and README, `version` and `vim_version` are reversed.  But each is correct.  Sorry for confusing you.

**Comment by MisanthropicBit** [comment]
Ah, I see. I have tried your suggestion and it works, the build still fails but the error is now related to my own test script :+1:.

Since it seems like `version` and `vim_version` are interchangeable, can I simply only use `vim_version` and not use `version` at all? Or is it better to use one as a backup as seems to be the case here and in other repositories I found that use the action? E.g. if `vim_version: 'latest'` does not exist fall back to `version: 'v8.2.0000'`.

> Certainly, in CI and README, version and vim_version are reversed. But each is correct. Sorry for confusing you.

No worries, I am new to Github Actions. Thanks for your help!
